cmd/agola/cmd/projectcreate.go

```diff
@@ -54,6 +55,7 @@ func init() {
 	flags.BoolVarP(&projectCreateOpts.skipSSHHostKeyCheck, "skip-ssh-host-key-check", "s", false, "skip ssh host key check")
 	flags.StringVar(&projectCreateOpts.parentPath, "parent", "", `parent project group path (i.e "org/org01" for root project group in org01, "user/user01/group01/subgroub01") or project group id where the project should be created`)
 	flags.StringVar(&projectCreateOpts.visibility, "visibility", "public", `project visibility (public or private)`)
+	flags.BoolVar(&projectCreateOpts.passVarsToForkedPR, "pass-vars-to-forked-pr", false, `pass variables to run even if triggered by PR from forked repo (default false)`)
```

Comment:
I'll make the option comment the same for this and project update. I'll just remove the part inside the brakets since the default and the type are automatically provided by cobra.